packages/jsii-pacmak/lib/targets/python.ts

```diff
@@ -42,6 +42,10 @@ const requirementsFile = path.resolve(
   'requirements-dev.txt',
 );
 
+// we use single-quotes for multi-line strings to allow examples within the
+// docstrings themselves to include double-quotes (see https://github.com/aws/jsii/issues/2569)
```

Comment:
That sounds very brittle to me... but okay...

Reply:
I believe this is sufficient to fix this p0.